src/taskpane/server.js

```diff
@@ -1923,48 +2313,105 @@ function sendExportEntriesGoogle(entriesForExport, sheetData, sheet, sheetRange,
+      }
+    }
+
+    // loop through comment objects to send
+    function sendSingleCommentEntry(j) {
+      if (!validations.includes(commentEntriesForExport[j].ArtifactId) && !(commentEntriesForExport[j].skip)) { //if this is a valid entry
+        var logResponse = manageSendingToSpira(commentEntriesForExport[j], model.user, model.currentProject.id, artifact, fields, fieldTypeEnums, log.parentId, isUpdate, true, false);
+        // update the parent ID for a subtypes based on the successful API call
+        if (artifact.hasSubType) {
+          log.parentId = logResponse.parentId;
+        }
+        log = processSendToSpiraResponse(j, logResponse, commentEntriesForExport, artifact, log, true, false);
+      }
     }
+
+    // loop through association objects to send
+    function sendSingleAssociationEntry(k) {
+      if (!validations.includes(associationEntriesForExport[k].SourceArtifactId) && !(associationEntriesForExport[k].skip)) { //if this is a valid entry
+        var logResponse = manageSendingToSpira(associationEntriesForExport[k], model.user, model.currentProject.id, artifact, fields, fieldTypeEnums, log.parentId, isUpdate, false, true)
+        // update the parent ID for a subtypes based on the successful API call
+        if (artifact.hasSubType) {
+          log.parentId = logResponse.parentId;
+        }
+
+        log = processSendToSpiraResponse(k, logResponse, associationEntriesForExport, artifact, log, false, true);
+      }
+    }
+
+
     // 4. SEND DATA TO SPIRA AND MANAGE RESPONSES
     // KICK OFF THE FOR LOOP (IE THE FUNCTION ABOVE) HERE
     // We use a function rather than a loop so that we can more readily use promises to chain things together and make the calls happen synchronously
     // we need the calls to be synchronous because we need to do the status and ID of the preceding entry for hierarchical artifacts
     for (var i = 0; i < entriesForExport.length; i++) {
-      if (!log.doNotContinue) {
-        log = checkSingleEntryForErrors(entriesForExport[i], log, artifact, isUpdate);
-        if (log.entries.length && log.entries[i] && log.entries[i].error) {
-          // do nothing
-        } else {
-          sendSingleEntry(i);
+      if (!entriesForExport[i].skip) {
+        if (!log.doNotContinue) {
+          log = checkSingleEntryForErrors(entriesForExport[i], log, artifact, isUpdate);
+          if (log.entries.length && log.entries[i] && log.entries[i].error) {
+            // do nothing
+          } else {
+            sendSingleEntry(i);
+          }
         }
       }
+      else {
+        var skip = {
+          "skip": true
+        };
+        log.entries.push(skip);
+        log.successCount++;
+
+      }
     }
 
+    //then, send comments for the artifact (if any)
+    for (var j = 0; j < commentEntriesForExport.length; j++) {
+      //make sure we don't have any error and we have the comment populated
+      if (Object.keys(commentEntriesForExport[j]).length > 1) {

```

Comment:
Do we need a check around here for if they are in advanced mode? Not sure - just asking

Reply:
The only way to have commentEntriesForExport populated is through Advanced Mode. I believe there's no need to re-check that flag.